src/services/cache/CacheService.ts

```diff
@@ -79,6 +92,123 @@ export class CacheService {
     return this.dmrService.readResource(request);
   }
 
+  getCacheStats(container: ICacheContainer, cache: ICache): ng.IPromise<any> {
+    let firstServer: IServerAddress = container.serverGroup.members[0];
+    let address: string [] = [].concat("host", firstServer.host, "server", firstServer.name, "subsystem",
+      "datagrid-infinispan", "cache-container", container.name, cache.type, cache.name);
```

Comment:
Minor, but let's make use of the generateAddress function here, like you have in `getCacheUsingType(..)` as this function will eventually return a different address when running in Standalone mode. The same applies to the other request addresses.